zfsbootmenu/libexec/zfsbootmenu-init

```diff
@@ -105,7 +105,10 @@ while true; do
 
   # shellcheck disable=SC2154
   if check_for_pools; then
-    if [ -n "${try_pool}" ]; then
+    if [ "${zbm_only_bpool:-0}" -eq 1 ]; then
```

Comment:
```suggestion
    if [ "${zbm_only_bpool}" = "only" ]; then
```
and also on line 133 in this diff ([line 130 in the original](https://github.com/zbm-dev/zfsbootmenu/blob/master/zfsbootmenu/libexec/zfsbootmenu-init#L130)) would replace
```sh
[ "${zbm_require_bpool:-0}" -ne 1]
```
with
```sh
[ -z "${zbm_require_bpool}" ]